GDSCommon-Demo/GDSCommon-Demo/Mocks/MockIntroViewModel.swift

```diff
@@ -0,0 +1,11 @@
+import GDSCommon
+import UIKit
+
+struct MockIntroViewModel: IntroViewModel {
+    var image: UIImage = UIImage(named: "badge") ?? UIImage()
+    var title: GDSLocalisedString = "This is a Welcome Screen"
```

Comment:
Think this mock text should be updated to `intro`

Reply:
Done 🙂